site/content/en/docs/Guides/feature-stages.md

```diff
@@ -35,7 +35,7 @@ The current set of `alpha` and `beta` feature gates are:
 {{% feature publishVersion="1.6.0" %}}
 | Feature Name | Gate    | Default | Stage | Since |
 |--------------|---------|---------|-------|-------|
-| Multicluster Allocation<sup>*</sup> | N/A | Enabled | `Alpha` | 0.11.0 |
+| Multicluster Allocation<sup>*</sup> | N/A | Enabled | `Stable` | 1.6.0 |
```

Comment:
@pooneh-m also confirming this change is good?

Reply:
How about keep the feature at the Beta stage for one more release? The APIs are at the stable version. Nothing is expected to change in terms of functionality between beta and stable. However, we mark the feature as a stable candidate, in case there are more feedback.

Also, please remove "Multicluster Allocation was started before this process was in place, and therefore is enabled by default"

Reply:
The footnote still partly applies. New features are supposed to have a feature gate and be disableable in beta, but since we started MCA before the process was in place it doesn't have a feature gate and can't be turned off. So maybe updating to "Multicluster Allocation was started before this process was in place, and therefore does not have a feature gate and cannot be disabled" would be the most accurate.

Reply:
@pooneh-m - I like keeping it at Beta - that's a good call. 
@roberthbailey - I also like that language, I've dropped that in.